src/gateways/BeaconsGateway.ts

```diff
@@ -36,4 +36,23 @@ export class BeaconsGateway implements IBeaconsGateway {
       throw Error(e);
     }
   }
+
+  public async saveBeacon(
+    beaconId: string,
+    updatedFields: Partial<IBeacon>
+  ): Promise<boolean> {
+    try {
+      const response = await axios.patch(
+        `${applicationConfig.apiUrl}/beacons/${beaconId}`,
+        updatedFields
+      );
+      return this.isSuccessful(response);
+    } catch (e) {
+      throw Error(e);
+    }
+  }
+
+  private isSuccessful(response: AxiosResponse) {
+    return 200 >= response.status && response.status <= 299;
+  }
```

Comment:
@stuart-madetech as discussed, to Keep It Simple I'm not using the returned updated object from the PATCH request here.